src/contributing/packaging.md

```diff
@@ -0,0 +1,38 @@
+# Packaging
+
+**NOTE:** This is not extensivly tested, and may break
+
+## Debian
+
+To package Darling for Debian-baded system we provide the ```tools/makedeb``` script.
```

Comment:
Inline code should use single backticks, triple backticks are for code blocks.

Reply:
Fixed